pkg/kubelet/apis/config/types.go

```diff
@@ -64,9 +64,15 @@ const (
 	// NoneTopologyManager Policy is a mode in which kubelet has no knowledge
 	// of NUMA alignment of a pod's CPU and device resources.
 	NoneTopologyManagerPolicy = "none"
-	// SingleNumaNodeTopologyManager Policy iis a mode in which kubelet only allows
+	// SingleNumaNodeTopologyManager Policy is a mode in which kubelet only allows
 	// pods with a single NUMA alignment of CPU and device resources.
 	SingleNumaNodeTopologyManager = "single-numa-node"
+	// ContainerTopologyScope represents that
+	// topology policy is applied on a per-container basis.
+	ContainerTopologyScope = "container"
```

Comment:
I would call this `ContainerTopologyManagerScope` and `PodTopologyManagerScope` to follow the pattern chosen for policies above.

Reply:
Done